code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -142,12 +142,14 @@
 				for (var/atom/movable/A as mob|obj in src)
 					A.forceMove(loc)
 					A.ex_act(severity++)
+					new /obj/item/stack/sheet/metal(loc)

```

Comment:
Err, this is going to spawn a new sheet of metal for every object/mob in the closet, that's probably not intentional.